ingestion/functions/parsing/peru/peru.py

```diff
@@ -0,0 +1,94 @@
+import json
+import os
+import sys
+from datetime import date, datetime
+import csv
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+if ('lambda' not in sys.argv[0]):
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'common/python'))
+import parsing_lib
+
+
+def convert_date(raw_date):
+    """ 
+    Convert raw date field into a value interpretable by the dataserver.
+
+    The date is listed in YYYYmmdd format, but the data server API will
+    assume that ambiguous cases (e.g. "05/06/2020") are in mm/dd/YYYY format.
+    
+    Adding line to ensure date has type str
+    """
+    raw_date = str(raw_date)
+    date = datetime.strptime(raw_date, "%Y%m%d")
+    return date.strftime("%m/%d/%Y")
+
+
+def convert_gender(raw_gender):
+    if raw_gender == "MASCULINO":
+        return "Male"
+    if raw_gender == "FEMENINO":
+        return "Female"
+    return None
+
+
```

Comment:
please run autopep8 to format your code as the other parsers (in vscode that's a simple extension to install), it will avoid having too much whitespace between functions like this